contrib/managed-azure/README.md

```diff
@@ -48,7 +48,7 @@ export HYPERSHIFT_BINARY_PATH="/path/go/src/github.com/openshift/hypershift/bin/
 export HYPERSHIFT_IMAGE="quay.io/hypershift/hypershift-operator:latest"
 export RELEASE_IMAGE="quay.io/openshift-release-dev/ocp-release:4.20.0-ec.3-multi"
 export LOCATION="eastus"
-export AZURE_CREDS="/path/azure-hypershift-dev.json"
+export AZURE_CREDS="/path/azure-creds.json"
```

Comment:
This is to match this line in docs and to unify the name. https://github.com/openshift/hypershift/blob/main/docs/content/how-to/azure/create-azure-cluster-on-aks.md?plain=1#L33